lib/api/api.go

```diff
@@ -1173,6 +1174,30 @@ func (s *service) getDBIgnores(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+func (s *service) getIgnoresFromDeletedFolder(w http.ResponseWriter, r *http.Request) {
+	qs := r.URL.Query()
+
+	path := qs.Get("path")
+	finalPath := path + "/.stignore"
+	if _, err := os.Stat(finalPath); !os.IsNotExist(err) {
+
+		file, err := os.Open(finalPath)
+		if err != nil {
+			log.Fatal(err)
+		}
+		defer file.Close()
+
+		b, err := ioutil.ReadAll(file)
```

Comment:
In fact, you can use `ioutil.ReadFile` to get all the file bytes directly without having to open+read+close. We typically call a byte slice `bs`.